assets/quick-add.css

```diff
@@ -35,6 +35,18 @@
   transform: none;
```

Comment:
Im still noticing the extra spacing at the bottom

https://screenshot.click/29-05-atpsx-c5big.mp4

Reply:
I think ideally the scrolling happening is only for the `.quick-order-list-container`. But that means quite a bit of rework of the CSS 🤔 
Cause the issue is the padding applied on `.quick-add-modal__content-info` I believe. 
You could keep the padding value applied for the left and right on `.quick-add-modal__content-info` and apply a top value on `.quick-add-modal__content-info--bulk-details` and then it might work as expected 👍

Reply:
@melissaperreault I thought it was the extra space when you scroll down?  🤔 

https://screenshot.click/29-05-atpsx-c5big.mp4

Reply:
Seems like @dan-menard fixed it. Issue was [this extra bit of scrolling](https://screenshot.click/03-06-jbj96-wwg23.mp4) at the bottom of the modal. 

👍